tools/alignment.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
Is it better to support alignment in the end of run.sh on test set? since different dataset may have different format data pipeline.